gm4_vigere_shamir/data/gm4_metallurgy/tags/functions/update_skull_owner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "values": [
+    "gm4_vigere_shamir:update_skull_owner"
+  ]
+}
```

Comment:
You don't need this file as your Shamir was created - and will therefore be merged - after PR #593. This is only required to update data on items created before that PR.